src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -279,6 +279,18 @@ protected override void ProcessRecord()
 
             if (UFormat != null)
             {
+                if (UFormat.Trim().Length == 0) 
```

Comment:
What about `null`? And I wonder why not use standard `[ValidateNotNullOrEmpty()]`?

Reply:
It seems that `[ValidateNotNullOrEmpty()]` does not take whitespace-only strings into account, so it looks like that has to be checked manually. As for `null`, it looks like the way the current code is written, if `$null` is passed to `-uformat`, it will just fall through the if-else completely and just print the date with a default format. Do we want to keep this behavior, or do we want to use [ValidateNotNull()]?

Reply:
" " - is allowed format string.
Disabling Null is breaking change - @SteveL-MSFT What is your thoughts?

Reply:
Thanks for the feedback. Just to clarify, I haven't changed the behavior of $null being passed in. I did not know that " " was a valid format string -- I assumed that this was also considered empty.

Reply:
Since `$null` is the same as if `-UFormat` wasn't provided, we probably don't want to break that behavior as someone may be relying on it.  `-UFormat` enables custom formatting, so any valid string should work.  I suppose it's not different if they provided `" "` or `"apple"` which should assume they know what they are doing.  So it seems that in this case, the only fix is on [this line](https://github.com/DdWr/PowerShell/blob/d7fc318c63e9192148fb100070b13deb9ea0266d/src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs#L349) if UFormat.Length == 0, just return.

Reply:
We can use `ValidateLength`

Reply:
@SteveL-MSFT @iSazonov Thank you for the responses. Based off this discussion, can we clarify the desired behavior? Based off of what has been said, this is the behavior it seems that we want here:

**Current**
* If `$null` is passed in, fall through and just print the date
* If `'  '`, `apple`, or any unconventional date format of length > 0 is passed in, keep the current behavior and show the formatted result.

**Proposed addition**
* Add a length check to [this line](https://github.com/DdWr/PowerShell/blob/d7fc318c63e9192148fb100070b13deb9ea0266d/src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs#L349), which will cause it to match the above behavior and format the date to an empty string.
* Use `ValidateLength`, which will throw an error if an empty string is passed in.

Reply:
I believe ` ValidateNotNullOrEmpty ` would best solution if @SteveL-MSFT'd agreed. This breaking change seems incredible.

Reply:
Updated PR with proposed changes.

Reply:
@SteveL-MSFT I think the uformat is static constant in most scenarios. If someone dynamically forms it, it's not likely that it will leave it uninitialized (null). So I believe this is a breaking change  only formally and we can accept it.

Reply:
I agree that most likely if $null gets passed, it's probably a problem in someone's script.  Let me get a quick ack from @PowerShell/powershell-committee before we take this breaking change.